MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -586,6 +591,17 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		2C67751E290AC4A6009C5EE4 /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				2C67751D290AC4A6009C5EE4 /* MapboxNavigationNative.xcframework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
```

Comment:
Did you intentionally changed that?

Reply:
I rolled it back. 

It was an attempt to fix the test run from `MapboxNavigation.xcodeproj`, but the problem with them was the lack of support for Carthage for `MapboxNavigation` target